src/tools/scripts/mailMensuelActivite/mailMensuelActivite.js

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env node
+'use strict';
+
+require('dotenv').config();
+
+const { execute } = require('../../utils');
+const dayjs = require('dayjs');
+const cli = require('commander');
+let { delay } = require('../../utils');
+const { getStatsDurees } = require('../../../../src/services/stats/cras/durees');
+const createEmails = require('../../../emails/emails');
+const createMailer = require('../../../mailer');
+
+cli.description('Envoi du mail mensuel d\'activité sur les CRAs')
+.option('--limit [limit]', 'limit the number of emails sent (default: 1)', parseInt)
+.option('--delai [delai]', 'Time in milliseconds to wait before sending the next email (default: 1000)', parseInt)
+.helpOption('-e', 'HELP command')
+.parse(process.argv);
+
+const listeMois = ['janvier', 'février', 'mars', 'avril', 'mai', 'juin', 'juillet', 'août', 'septembre', 'octobre', 'novembre', 'décembre'];
+const mois = new Date(dayjs(Date.now()).subtract(1, 'month'));
+const debutMois = new Date(dayjs(mois).startOf('month'));
+const finMois = new Date(dayjs(mois).endOf('month'));
+
+const moisDernier = new Date(dayjs(Date.now()).subtract(2, 'month'));
+const debutMoisDernier = new Date(dayjs(moisDernier).startOf('month'));
+const finMoisDernier = new Date(dayjs(moisDernier).endOf('month'));
+
+const getConseillers = db => async limit => await db.collection('conseillers').find({
+  'emailCN.address': { $exists: true },
+  'estCoordinateur': { $ne: true },
+}).limit(limit).toArray();
+
+const getCountCrasMois = db => async query => await db.collection('cras').countDocuments(query);
+
+const getCountCrasMoisDernier = db => async query => await db.collection('cras').countDocuments(query);
+
+const getNbUsagersMois = db => async query => await db.collection('cras').aggregate(
+  {
+    $match:
+    { ...query }
+  },
+  { $group:
+    { _id: null, count: { $sum: '$cra.nbParticipants' } }
+  },
+  { $project: { 'valeur': '$count' } }
+).toArray();
+
+const getNbUsagersMoisDernier = db => async query => await db.collection('cras').aggregate(
+  { $match:
+    { ...query }
+  },
+  { $group:
+    { _id: null, count: { $sum: '$cra.nbParticipants' } }
+  },
+  { $project: { 'valeur': '$count' } }
+).toArray();
+
+const getMoins1Heure = async arrayHeures => {
+  let totalMinutes = 0;
+  arrayHeures.forEach(heures => {
+    if (heures.nom === '0-30') {
+      totalMinutes += 30 * heures.valeur;
+    } else if (heures.nom === '30-60') {
+      totalMinutes += 60 * heures.valeur;
+    }
+  });
+  return Math.round(totalMinutes / 60);
+};
+const getPlus1Heure = async arrayHeures => {
+  let totalMinutes = 0;
+  arrayHeures.forEach(heures => {
+    totalMinutes += heures._id * heures.count;
+  });
+  return Math.round(totalMinutes / 60);
+};
+
+const get1hEtPlusMois = db => async query => await db.collection('cras').aggregate(
+  { $unwind: '$cra.duree' },
+  { $match: { ...query,
+    'cra.duree': { $gte: 60 } } },
```

Comment:
je sais pas si ca impact : fichier src\services\stats\cras\durees.js
 'cra.duree': { $in: ['60', '90'] } } //Correspond au bouton 1h pile et 1h 30 (60 cra v1 & 90 cra v2)

Reply:
à partir de 60min on peut choisir au quart d'heure près et ce n'est plus une tranche d'horaires, mais un nombre (en string)